Frontend/Pages/TaskList/HtbDocument/Preview.cshtml.cs

```diff
@@ -16,7 +16,7 @@ public class Preview : ProjectPageModel
         public string ProjectUrn => Project.Urn;
         public object OutgoingAcademyUrn => TransferringAcademy.Urn;
 
-        public FeaturesSummaryViewModel FeaturesSummaryViewModel { get; set; }
+        public Index FeaturesSummary { get; set; }
```

Comment:
Might be best keeping the names consistent with the other ones but tbh this whole class prob needs some refactoring so we can look at then

Reply:
Good call, restored name nere